### PR TITLE
[Conditional mark] Remove xfail mark of warm reboot cases

### PR DESCRIPTION
**Pull request by Gfrom2016**
### Description of PR

Summary:
Fixes
Remove xfail marks on warm-reboot cases since Broadcom has fix for it.

### Type of change

- [ ] Bug fix
- [ ] Testbed and Framework(new/improvement)
- [ ] Test case(new/improvement)

### Back port request
- [ ] 201911
- [ ] 202012
- [ ] 202205
- [x] 202305

#### What is the motivation for this PR?
Enable warm-reboot cases.
Workitem: 26255229
#### How did you do it?
Remove xfail conditional mark on:
- pfcwd/test_pfcwd_warm_reboot.py
- platform_tests/test_cont_warm_reboot.py
- arp/test_wr_arp.py

### CODE REVIEW

File: tests/common/plugins/conditional_mark/tests_mark_conditions.yaml [hunk 1]
```diff
@@ -870,10 +866,6 @@ platform_tests/test_auto_negotiation.py:
     reason: "auto negotiation test highly depends on test enviroments, file issue to track and skip for now"
     conditions: https://github.com/sonic-net/sonic-mgmt/issues/5447
 
-platform_tests/test_cont_warm_reboot.py:
```
Comment (StormLiangMS) [inline tests/common/plugins/conditional_mark/tests_mark_conditions.yaml:873]: should we xfail this one on dualtor?

**Review by StormLiangMS — APPROVED** [review]
approved with minor suggestion.

File: tests/common/plugins/conditional_mark/tests_mark_conditions.yaml (hunk 1 above)
Comment (Gfrom2016) [reply tests/common/plugins/conditional_mark/tests_mark_conditions.yaml:873]: skip this on dualtor

**Review by StormLiangMS — APPROVED** [review]
LGTM

**Pull request merged**